### PR TITLE
Aea 1467 dockerfile

### PR DESCRIPTION
**Pull request by callumtipper-IW**

### CODE REVIEW

**Comment by booshibooshi** [comment]
Converted to draft until we have a 200 response from `_status` endpoint so we're not spaming alerts from monitoring

File: ecs-proxies-deploy.yml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+          - name: DEMO_APP_KEY_ID
+            valueFrom: "/ptl/api-deployment/eps-api-tool/test-client/local-jwt-key-id"
+          - name: RSS_JWT_ISSUER
+            valueFrom: "/ptl/api-deployment/eps-api-tool/test-client/rss-jwt-issuer"
+          - name: RSS_JWT_KID
+            valueFrom: "/ptl/api-deployment/eps-api-tool/test-client/rss-jwt-key-id"
+          - name: RSS_JWT_SUBJECT
+            valueFrom: "/ptl/api-deployment/eps-api-tool/test-client/rss-jwt-subject"
+          - name: RSS_JWT_PRIVATE_KEY
+            valueFrom: "ptl/eps-api-tool/test-client/rss-jwt-private-key"
```
Comment (booshibooshi) [inline ecs-proxies-deploy.yml:43]: some secrets are `/ptl/` some are `ptl/`, ensure these are consistent

File: ecs-proxies-deploy.yml [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+          - name: DEMO_BASE_PATH
+            value: "https://{{ APIGEE_HOSTNAME }}/{{ SERVICE_BASE_PATH }}"
+          - name: EPS_BASE_PATH
+            value: "https://int.api.service.nhs.uk/electronic-prescriptions/FHIR/R4"
+          - name: OAUTH_BASE_PATH
+            value: "https://int.api.service.nhs.uk/oauth2"
+          - name: OAUTH_REDIRECT_URI
+            value: "https://{{ APIGEE_HOSTNAME }}/{{ SERVICE_BASE_PATH }}/callback"
+          - name: SIGNING_BASE_PATH
+            value: "https://int.api.service.nhs.uk/signing-service"
```
Comment (booshibooshi) [inline ecs-proxies-deploy.yml:30]: https://{{ APIGEE_HOSTNAME }}/signing-service

File: ecs-proxies-deploy.yml [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+          - name: ENVIRONMENT
+            value: "{{ APIGEE_ENVIRONMENT }}"
+          - name: DATABASE_URL
+            value: "PLACEHOLDER"
+          - name: DEMO_BASE_PATH
+            value: "https://{{ APIGEE_HOSTNAME }}/{{ SERVICE_BASE_PATH }}"
+          - name: EPS_BASE_PATH
+            value: "https://int.api.service.nhs.uk/electronic-prescriptions/FHIR/R4"
+          - name: OAUTH_BASE_PATH
+            value: "https://int.api.service.nhs.uk/oauth2"
```
Comment (booshibooshi) [inline ecs-proxies-deploy.yml:26]: https://{{ APIGEE_HOSTNAME }}/oauth2

File: ecs-proxies-deploy.yml [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+          - name: COMMIT_ID
+            value: "{{ SOURCE_COMMIT_ID }}"
+          - name: ENVIRONMENT
+            value: "{{ APIGEE_ENVIRONMENT }}"
+          - name: DATABASE_URL
+            value: "PLACEHOLDER"
+          - name: DEMO_BASE_PATH
+            value: "https://{{ APIGEE_HOSTNAME }}/{{ SERVICE_BASE_PATH }}"
+          - name: EPS_BASE_PATH
+            value: "https://int.api.service.nhs.uk/electronic-prescriptions/FHIR/R4"
```
Comment (booshibooshi) [inline ecs-proxies-deploy.yml:24]: https://{{ APIGEE_HOSTNAME }}/electronic-prescriptions/FHIR/R4

File: ecs-proxies-deploy.yml (hunk 1 above)
Comment (booshibooshi) [reply ecs-proxies-deploy.yml:43]: also might want to namespace these per environment? `dev| int | prod`

Comment (booshibooshi) [reply ecs-proxies-deploy.yml:43]: can replace `ptl` with `{{ account }}` for non-live/live secrets

Comment (callumtipper-IW) [reply ecs-proxies-deploy.yml]: Changed the {{account}} part, not sure how to namespace to dev|int|prod

File: ecs-proxies-deploy.yml (hunk 2 above)
Comment (callumtipper-IW) [reply ecs-proxies-deploy.yml]: done

File: ecs-proxies-deploy.yml (hunk 3 above)
Comment (callumtipper-IW) [reply ecs-proxies-deploy.yml]: done

File: ecs-proxies-deploy.yml (hunk 4 above)
Comment (callumtipper-IW) [reply ecs-proxies-deploy.yml]: done

File: ecs-proxies-deploy.yml (hunk 1 above)
Comment (booshibooshi) [reply ecs-proxies-deploy.yml]: have updated to namespace per environment

**Comment by booshibooshi** [comment]
Waiting on PR to be merged to infrastructure

**Pull request closed without merging**